lib/rummager/app.rb

```diff
@@ -318,17 +318,17 @@ def get_type_from_request_body(request_body)
   end
 
   def serve_from_s3(key)
-    o = Aws::S3::Object.new(bucket_name: ENV["AWS_S3_SITEMAPS_BUCKET_NAME"], key:)
+    o = Services.s3_client.get_object(bucket: ENV["AWS_S3_SITEMAPS_BUCKET_NAME"], key:)
```

Comment:
Would you mind linking to the AWS docs in this commit that shows they're functionally equivalent? I tried to dig it out but the docs are 😱